docs/stdlib/datetime.rst

```diff
@@ -369,29 +369,28 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         ...        <cal::relative_duration>'1 hour';
         {<cal::local_time>'23:00:00'}
 
-    If an arithmetic operation results in a day that doesn't exist in the given
-    month, the last day of the month is used instead.
+    If an arithmetic operation results in a day that is nonexistent in the
+    given month, the last day of the month will be used instead:
```

Comment:
Keep the "will be" and the colon but revert the rest. "doesn't exist" to "is nonexistent" doesn't add clarity or improve readability.